app/Http/Controllers/Service/Admin/VouchersController.php

```diff
@@ -47,20 +50,63 @@ public function create()
      */
     public function storeBatch(Request $request)
     {
-        $new_vouchers = [];
+        $newVouchers = [];
+
+        $sponsorIds = \App\Sponsor::all()->pluck('id')->toArray();
+
+        $voucherRules = [
+            'sponsor_id' => [
+                'required',
+                Rule::in($sponsorIds)
```

Comment:
My reading if exists is that it checks if a column header exists in DB. Not sure it actually checks the values therein. A prepared to be wrong though!